extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -737,6 +738,8 @@ Check the following table to see the mapping of Adyen [result codes](https://doc
 | Refused| Authorization (Failure)|
 | Error| Authorization (Failure)|
 
+After resolving the issue on the refused payment request, to be able to retry the request existing payment should not be reused, instead a new commercetools payment resource needs to be created and added to the appropriate cart, please refer [creating a commercetools payment](#step-2-creating-a-commercetools-payment) section for more details.
+
```

Comment:
I guess we can skip it if we use the suggested text above.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/646/commits/7bd98ae20f6a163a178b319106d9f84177815e9f